### ISSUE TITLE
Top dendrogram hover in the air above the Clustergram heatmap

### ISSUE THREAD

**Issue by joelostblom** [plotly/dash-bio#448]
There is currently a large amount of empty space between the top dendrogram and the heatmap. This can lead the reader to believe that there is something missing in the empty space. I suggest anchoring the dendrograms to the plot and only shifting them upwards if there are any labels specified that needs to go between the dendrogram and the plot.

```python
import dash_bio as db
import plotly.express as px

iris = px.data.iris()
db.Clustergram(iris.select_dtypes('number').to_numpy())
```
Current appearance:
![image](https://user-images.githubusercontent.com/4560057/69484171-61c8f380-0e30-11ea-8f2d-8ea31314e876.png)

Suggested appearance:
![image](https://user-images.githubusercontent.com/4560057/69485699-4ddabd00-0e43-11ea-918d-93042f72a795.png)

```
-----
dash      	1.6.1
dash_bio  	0.4.4
dash_core_components	1.5.1
dash_html_components	1.0.2
numpy     	1.17.3
pandas    	0.25.3
plotly    	4.3.0
-----
IPython   	7.9.0
jupyter_client	5.3.3
jupyter_core	4.6.1
notebook  	6.0.1
-----
Python 3.8.0 | packaged by conda-forge | (default, Nov 22 2019, 19:11:38) [GCC 7.3.0]
```

**Comment by josham** [comment]
Not ideal but you can adjust the domain manually (`dash-bio==0.4.8`):

```python
import dash_bio as db
import plotly.express as px

iris = px.data.iris()
fig = db.Clustergram(
    iris.select_dtypes('number').to_numpy(),
    hidden_labels=['row', 'col']
)

fig.update_layout(
    yaxis2={'domain': [0.74, 1]}
)
```
![cluster](https://user-images.githubusercontent.com/3686522/77856128-071b1c00-71c3-11ea-9967-268e67068868.png)

**Comment by ijmiller2** [comment]
This is something I'd like to see addressed more systematically too. But @josham's suggestion appears to have worked for me.

**Comment by kinimesi** [comment]
Looks like this has been fixed a while ago with ba8ce1071d85da0f10c76a19253f07593cc09a4b I can't reproduce it with the current version of dash-bio. Closing, please reopen if the issue persists. Thanks.
